src/main/resources/assets/apotheosis/lang/en_us.json

```diff
@@ -87,6 +87,7 @@
 
     "death.attack.apotheosis:execute": "%s was executed by %s",
```

Comment:
@Shadows-of-Fire This file has 2 same keys `death.attack.apotheosis:execute`, but they have different values, the one is `%s was executed by %s`, the other is `%s was executed`, who is correct?